internal/kafka/test/integration/kafkas_test.go

```diff
@@ -1313,7 +1313,7 @@ func TestKafkaDelete_DeleteDuringCreation(t *testing.T) {
 		if ocmConfig.MockMode == ocm.MockModeEmulateServer {
 			// increase repeat interval to allow time to delete the kafka instance before moving onto the next state
 			// no need to reset this on teardown as it is always set at the start of each test within the registerIntegrationWithHooks setup for emulated servers.
-			workers.RepeatInterval = 10 * time.Second
+			workers.NewReconcilerConfig().RepeatInterval = 10 * time.Second
```

Comment:
Hi @machi1990, I am unsure about this call, similarly with the call in helper.go